packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -156,6 +157,29 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
     super(context);
 
     this.helpers = new Helpers(context.options);
+
+    let indentWidth = context.options.indentWidth;
```

Comment:
This is the only thing that jumps out to me as a possible source of where some of these failures are coming from: Since it's `context.options` instead of `this.context.options`, something may be getting adjusted in the `super` call that is hammering everything else.